seeds/balancer/balancer_transfers_bpt_seed.csv

```diff
@@ -0,0 +1,36 @@
+blockchain,contract_address,evt_tx_hash,evt_index,evt_block_time,block_date,evt_block_number,from,to,value
+arbitrum,0xfb5e6d0c1dfed2ba000fbc040ab8df3615ac329c,0xe8e94ad465e17ae5f4e667f686b3e0f2a1d07da8b58407ce9bf2b770246725d5,0,2022-12-05T14:41:08Z,2022-12-05T00:00:00Z,43831333,0x7158ee69dcfdfbf2118b5a2cfb8fb2e82d77e169,0x5b6776cd9c51768fc915cad7a7e8f5c4a6331131,1.0893485052912E+26
+arbitrum,0xfb5e6d0c1dfed2ba000fbc040ab8df3615ac329c,0xfc0459a6cf60b92903362085dcdef01fec1aa706e2df6423758b2bc67c6fcc6d,3,2022-12-05T11:35:06Z,2022-12-05T00:00:00Z,43788566,0x0000000000000000000000000000000000000000,0xce88686553686da562ce7cea497ce749da109f9f,1.6313527900000E+16
+arbitrum,0xfb5e6d0c1dfed2ba000fbc040ab8df3615ac329c,0x8ba25dd96f57b60c98372473a79d3d4e2e23767cec4fe1bfb990179f89669e89,2,2022-12-05T14:03:14Z,2022-12-05T00:00:00Z,43822509,0x0000000000000000000000000000000000000000,0x481787bba8c56f801db8bdb336844b771c011246,2.1517242877094E+26
+arbitrum,0xfb5e6d0c1dfed2ba000fbc040ab8df3615ac329c,0x8b9c2c80ab9ca58cd90866c8b08f2d21369b870243647bbfd0e55946ccb09ecf,0,2022-12-05T14:22:43Z,2022-12-05T00:00:00Z,43827069,0xc1739ad437c66fa06fc16952db85fa950cc7cd0c,0x5b6776cd9c51768fc915cad7a7e8f5c4a6331131,1.0893461527000E+21
+arbitrum,0xfb5e6d0c1dfed2ba000fbc040ab8df3615ac329c,0x9cc6bd5eef1911ba12fb5ed2e374ff5dc7eaa53d19f0dbe4a5d2d23f40fd5eb6,1,2022-12-05T21:33:28Z,2022-12-05T00:00:00Z,43915646,0x0000000000000000000000000000000000000000,0xce88686553686da562ce7cea497ce749da109f9f,3518445.0
+ethereum,0x10a2f8bd81ee2898d7ed18fb8f114034a549fa59,0xe09414467ec16e798a2c9865aba6a59069888331ede90b1bb9e4d2592ad910b1,495,2022-04-14T09:51:36Z,2022-04-14T00:00:00Z,14582842,0x0000000000000000000000000000000000000000,0x654be0b5556f8eadbc2d140505445fa32715ef2b,0.0
+ethereum,0x8f4205e1604133d1875a3e771ae7e4f2b0865639,0x4a3d883586a4f9103655ffe48bb6112c440b781bfcd7b9cefaa5ab920b451962,261,2022-04-14T17:58:59Z,2022-04-14T00:00:00Z,14585043,1.07215212476546E+62,0xc43d32bc349cea7e0fe829f53e26096c184756fa,4.2973128445430E+35
+ethereum,0x8f4205e1604133d1875a3e771ae7e4f2b0865639,0x4b6ff446bde29b8dcf3a341260347bfa5c2e91f3f7262781272b7bb673f03300,103,2022-04-14T08:47:47Z,2022-04-14T00:00:00Z,14582537,1.07215212476546E+62,0xc43d32bc349cea7e0fe829f53e26096c184756fa,1.0424721680068E+36
```

Comment:
Hi @Stefenon, these 2 lines having addresses contain `E+62`, which seems illegal. I will remove these 2 lines first.

Reply:
Hey @Hosuke, the test are still not successful. I believe it's a problem with the comma after selecting the `model_sample.value` column at [line 130](https://github.com/duneanalytics/spellbook/actions/runs/3799062071/jobs/6461247555#step:10:131). This error also appears in lines [300](https://github.com/duneanalytics/spellbook/actions/runs/3799062071/jobs/6461247555#step:10:301), [470](https://github.com/duneanalytics/spellbook/actions/runs/3799062071/jobs/6461247555#step:10:471) and [640](https://github.com/duneanalytics/spellbook/actions/runs/3799062071/jobs/6461247555#step:10:641). Is it a problem with our seeds or how we configured the tests?

Reply:
I think the problem may be caused by the settings in the schema.yml file. 
Like this https://github.com/duneanalytics/spellbook/pull/2056/files#diff-45e8d0b4bb1727e1c0fa010b34893a9085153714cdd2d2b8b8b42c017b324dd5R100-R109
I will fix this soon.